src/mmrelay/meshtastic_utils.py

```
@@ -3944,28 +4102,71 @@ def connect_iface(iface_param: Any) -> None:
                         )
                         try:
-                            if meshtastic_client is meshtastic_iface:
+                            if client is meshtastic_iface:
                                 # BLE interface - use proper disconnect sequence
                                 _disconnect_ble_interface(
                                     meshtastic_iface, reason="address validation failed"
                                 )
                             else:
-                                meshtastic_client.close()
+                                client.close()
                         except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)